source/jormungandr/tests/line_sections_tests.py

```diff
@@ -35,9 +35,9 @@
 
 def impacted_ids(disrupts):
     # for the impacted obj, either get the id, or the headsign (for the display_information field)
```

Comment:
minor: maybe more details on comment, but... "Looking for (identifiable) objects impacted by 'disruptions' inputed" (so then, rename parameter)